packages/@contentful--app-scripts/lib/cache-credential/index.test.js

```diff
@@ -0,0 +1,69 @@
+const { cacheEnvVars } = require('./index');
+const fs = require('fs');
+const assert = require('assert');
+
+const fsPromises = fs.promises;
+
+describe('Caching environemnt variables', () => {
+  before(async () => {
+    try {
+      await fsPromises.access('.env', fs.constants.F_OK);
+      await fsPromises.copyFile('.env', '.env.temp');
```

Comment:
What happens here? Where does this file come from?

Reply:
Just in case you have a `.env` lying around. Unlikely but can't be too careful 🤷